.github/workflows/ci.yml

```
@@ -2,19 +2,28 @@ name: CI Build and Test
 
 on:
   push:
-    branches: [ "main", "master" ]
+    branches: [ "main", "development" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)